DFC.ServiceTaxonomy.CompUi/Handlers/CacheHandler.cs

```diff
@@ -81,7 +81,7 @@ public async Task ProcessPublishedAsync(PublishContentContext context)
         //Temp check to see if the published items content type matches any of the allowed items in the list
         var pageRouteFlag = PageRouteFlag(processing);
 
-        if (pageRouteFlag && eventGridContentTypes.Any(contentType => context.ContentItem.ContentType.Contains(contentType)))
+        if (pageRouteFlag && eventGridContentTypes.Any(contentType => context.ContentItem.ContentType.ToLower().Contains(contentType)))
```

Comment:
May be better to use `.Contains(contentType, StringComparison.OrdinalIgnoreCase)`, then you don't have to think of the casing for any of the items.